### PR TITLE
Fix several issues:

### LINKED ISSUE #104: ETQA abbreviations (and other lookups) need to be indexed for search
I can search successfully for "media, information and communication" but not for MICTSETA. I think searches for courses under a SETA/ETQA will tend to use the abbreviation. 

btw, search clauses in " " are treated as one search term which is a great feature.

### LINKED ISSUE #114: Please expose the lookup tables as inlines in django admin

### PR DESCRIPTION
**Pull request by lucernae**
- fix #114 : Admin inline for address in campus admin
- fix #104 : Include ETQA abbreviation in the index fix #104
- issue #48 : Add features to search. Now can use quotation like "Cape Town" and using 'AND' operator in a query
  So, if you search : "Cape Town" engineering  --> will return any providers that include Cape Town 'and' with engineering courses
  If you search : "Cape Town" -engineering --> This time, it will exclude engineering courses

### CODE REVIEW

File: deployment/Makefile [hunk 1]
```diff
@@ -229,6 +229,27 @@ staging-migrate:
 	@echo "------------------------------------------------------------------"
 	@docker-compose -p $(PROJECT_ID)-staging run uwsgi python manage.py migrate
 
+staging-updatelongdescription:
```
Comment (cchristelis) [inline deployment/Makefile]: We need a non-staging command that does this too.

**Comment by cchristelis** [comment]
Hi @lucernae,

I tried to search for Cape Town Engineering and got the following result:
    Belcomec Training
    National Certificate in Electrical Engineering - Culture, Arts, Tourism, Hospitality and Sports Sector Education and Training Authority (CATHSSETA)

When I searched for only Cape Town I did not get this result. I don't expect there to be more results when I add terms, but rather less with an AND. Am I missing something?

File: deployment/Makefile (hunk 1 above)
Comment (lucernae) [inline deployment/Makefile]: Hey, that's right. I will note this

**Comment by lucernae** [comment]
@cchristelis did you do reindex?
If you search Cape Town, you should have results with any providers which possibly got Cape Town in the index (from the name or perhaps address). 
Then if you search Cape Town Engineering, you should have less results.

To be sure, I will merge this first and deploy it to staging.

**Comment by cchristelis** [comment]
@lucernae, I am getting 504's on the staging site, since you merged...

**Pull request merged**